### PR TITLE
[8.1] Production System: from DN to Author

### PR DESCRIPTION
**Pull request by fstagni**
- [x] update wiki

BEGINRELEASENOTES

*Production
CHANGE: Move from DN to Username

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review OK?

File: src/DIRAC/ProductionSystem/DB/ProductionDB.sql [hunk 1]
```diff
@@ -25,7 +25,8 @@ CREATE TABLE Productions(
     Description LONGTEXT,
     CreationDate DATETIME,
     LastUpdate DATETIME,
-    AuthorDN VARCHAR(255) NOT NULL,
+    Author VARCHAR(255) NOT NULL,
+    AuthorDN VARCHAR(255),
```
Comment (andresailer) [inline src/DIRAC/ProductionSystem/DB/ProductionDB.sql]:
```suggestion
    AuthorDN VARCHAR(255) DEFAULT NULL,
```
Wasn't there a problem with the transformationDB about no default value for AuthorDN?

Comment (fstagni) [reply src/DIRAC/ProductionSystem/DB/ProductionDB.sql]: Does not seem so: https://github.com/DIRACGrid/DIRAC/pull/7124/files#diff-c9f689c0427d70b7ff70c68598d409ab7d16f597e4fe484e24627ab893bda479

Comment (andresailer) [reply src/DIRAC/ProductionSystem/DB/ProductionDB.sql]: In the hackathon on August 3rd there was a problem about AuthorDN not having a default value, wasn't there?

Comment (fstagni) [reply src/DIRAC/ProductionSystem/DB/ProductionDB.sql]: I don't remember about AuthorDN issues for TrasformationSystem, but there was a similar thing for WMS, for which I created https://github.com/DIRACGrid/DIRAC/pull/7155/files

Comment (andresailer) [reply src/DIRAC/ProductionSystem/DB/ProductionDB.sql]: On the certif DB for TransformationDB.Transformations: 
```
`AuthorDN` varchar(255) DEFAULT NULL,`
```

File: src/DIRAC/ProductionSystem/Service/ProductionManagerHandler.py [hunk 2]
```diff
@@ -34,9 +34,9 @@ def initializeHandler(cls, serviceInfoDict):
 
     def export_addProduction(self, prodName, prodDescription):
         credDict = self.getRemoteCredentials()
-        authorDN = credDict.get("DN", credDict.get("CN"))
+        author = credDict.get("username")
```
Comment (chaen) [inline src/DIRAC/ProductionSystem/Service/ProductionManagerHandler.py]: having no author at all seems off no ?

**Pull request merged**